readme/continuous-integration/release-notification.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Release Notification
+---
+
+Once FeatureMap has been integrated into an application, the knowledge of which files are assigned to a given Feature may be used during the publication of a release (or upon the merge to a main branch).  The specific implementation depends largely on the structure of a given application, but broadly you may consider:
+
+  - Inspect the files that have been modified as part of a given release
+  - Use FeatureMap's [`for_file`](({{ '/interface/for-file' | relative_url }})) public interface to retrieve Feature assignment data for each of those files
+  - Collect and group changes to each associated Feature
```

Comment:
It might be helpful to highlight the public `group_commits` ([source](https://github.com/Beyond-Finance/feature_map/blob/8592afe515649ccf936b539e9419070c6daced7f/lib/feature_map.rb#L212)) and `generate_release_notification` ([source](https://github.com/Beyond-Finance/feature_map/blob/8592afe515649ccf936b539e9419070c6daced7f/lib/feature_map.rb#L243C7-L243C36)) methods as utilities that can make this process easier to setup and manage.